.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
> For the moment, yes. But with fast_finish enable this would not be a problem.

I mean confusing for someone who would read the travis config. I have never seen such usage.
I think it's premature optimization ATM.